### PR TITLE
Valet reservation refactoring

### PR DESCRIPTION
**Pull request by kjmak14**
No functional differences. Changes logic in the fetch for spots and reservations to use a promise.All instead of async await. Included time as a state. Cleaned up some console.logs

### CODE REVIEW

**Review by jose-m-f — APPROVED** [review]
Thanks for cleaning up the logs. This looks great

File: src/components/valet-reservation/ReservationList.jsx [hunk 1]
```diff
@@ -29,9 +23,10 @@ const ReservationList = () => {
       </button>
       <ul>
         {filterList.map((reservation) => {
```
Comment (bzw24) [inline src/components/valet-reservation/ReservationList.jsx:25]: delete the console log

**Review by bzw24 — APPROVED** [review]
Good with all the changes

File: src/components/valet-reservation/ReservationList.jsx (hunk 1 above)
Comment (kjmak14) [reply src/components/valet-reservation/ReservationList.jsx:25]: done

**Pull request merged**